docs/develop/integrations/vercel-integration.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 7
+sidebar_label: Vercel
+title: Integrating Momento with your Vercel app
+description: Learn to deploy a chat app using Momento Javascript Web SDK to Vercel.
+pagination_next: null
+---
+
+[Vercel](https://vercel.com/) is a popular platform for building, deploying, and scaling modern websites and apps. Vercel are also the creators and maintainers of [Next.js](https://nextjs.org/). Their platform's offerings include a fast global edge network, the scalability of serverless, and deployments seamlessly integrated with the most popular Git providers: [GitHub](https://github.com/), [GitLab](https://gitlab.com), and [Bitbucket](https://bitbucket.org/). 
+
+In this tutorial, we'll make use of a simple example [Next.js chat application](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/web/nextjs-chat) from our [Javascript SDK](https://github.com/momentohq/client-sdk-javascript). By the end of the exercise, you'll have a realtime browser-based chat application up and running at a URL you can use to invite your friends to the chat.
+
+## Prerequisites
+
+To get this app deployed and running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- A [Vercel](https://vercel.com/) account.
+- A copy or fork of the [Momento Javascript SDK](https://github.com/momentohq/client-sdk-javascript) in your personal repositories.
+- A Momento authentication token.
+- A Momento cache.
```

Comment:
Is this created in the console or programmatically? If console, this should probably be stated and a link to the console provided.